timesketch/frontend-ng/src/components/LeftPanel/ThreatIntel.vue

```diff
@@ -0,0 +1,238 @@
+<!--
+Copyright 2023 Google Inc. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<template>
+  <div>
+    <v-row
+      no-gutters
+      style="cursor: pointer"
+      class="pa-4"
+      flat
+      @click="expanded = !expanded"
+      :class="$vuetify.theme.dark ? 'dark-hover' : 'light-hover'"
+    >
+      <span
+        ><v-icon left>mdi-shield-search</v-icon> Threat Intelligence (<small
+          ><strong>{{ intelligenceData.length }}</strong></small
+        >)</span
+      >
+      <v-spacer></v-spacer>
+
+      <v-btn
+        v-if="expanded"
+        small
+        depressed
+        color="primary"
+        outlined
+        :to="{ name: 'Intelligence', params: { sketchId: sketch.id } }"
+        @click.stop=""
+      >
+        <v-icon small left>mdi-pencil</v-icon>Manage
+      </v-btn>
+    </v-row>
+
+    <v-expand-transition>
+      <div v-show="expanded">
+        <v-divider></v-divider>
+
+        <v-tabs grow v-model="tabs">
+          <v-tab
+            >Indicators (<small>{{ intelligenceData.length }}</small
+            >)</v-tab
+          >
+          <v-tab
+            >Tags (<small>{{ Object.keys(tagInfo).length }}</small
+            >)</v-tab
+          >
+        </v-tabs>
+        <v-tabs-items v-model="tabs">
+          <v-tab-item :transition="false">
+            <v-data-table dense :headers="indicatorHeaders" :items="intelligenceData" :items-per-page="10">
+              <template v-slot:item.ioc="{ item }">
+                <span v-if="item.type === 'hash_sha256'" :title="item.ioc">
+                  <span>{{ item.ioc.substring(0, 8) }}...{{ item.ioc.substring(item.ioc.length - 8) }}</span>
```

Comment:
Instead of this, could we used a fixed-width column size and text-overflow set to ellipsis? https://www.w3schools.com/cssref/css3_pr_text-overflow.php

Reply:
Yes we can. It's common to "human pattern match" hashes on beginning and end though. So I thought we would make that easier. WDYT?